packages/sommelier.rb

```diff
@@ -107,7 +151,8 @@ def self.postinstall
     puts "Please be aware that gui applications may not work without the sommelier daemon running.".orange
     puts
     puts "If you are upgrading from an earlier version of sommelier, run".orange
-    puts "'stopsommelier && startsommelier' or logout and login again.".orange
+    puts "'stopsommelier && restartsommelier' or logout and login again.".orange
```

Comment:
Should only need `restartsommelier`, right?

Reply:
restartsommelier is an alias which requires reloading .bashrc  - maybe that and startsommelier should be separate shellscripts, since these aliases are fragile?

Reply:
I guess we can test and find out.

Reply:
Those have all been moved to separate scripts, with initsommelier renamed to startsommelier.